admin/src/components/UI/Button/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import "./style.scss";
+
+export default function Button(props) {
+  return (
+    <button className='button' onClick={props.onClick}>
+      {props.title}
+    </button>
```

Comment:
```suggestion
export default function Button({ children, onClick, ...rest }) {
  return (
    <button className='button' onClick={onClick} {...rest}>
      {children}
    </button>
```
Như này rõ ràng và đầy đủ hơn nhé
Trong trường hợp trong button có icon thì hoàn toàn có thể truyền vào
Ngoài ra truyền {...rest} để còn có thể truyền các attr khác
VD:
`<Button type='submit'></Button>`